plugins/woocommerce/client/legacy/js/frontend/checkout.js

```diff
@@ -824,6 +825,19 @@ jQuery( function( $ ) {
 				},
 				dataType: 'html'
 			});
+		},
+		/**
+		 * Handle when pressing the Space key on the remove coupon link.
+		 * This is necessary because the link got the role="button" attribute
+		 * and needs to act like a button.
+		 *
+		 * @param {Object} e The JQuery event
+		 */
+		on_keydown_remove_coupon: function( e ) {
+			if ( e.key === ' ' ) {
+				e.preventDefault();
+				$( this ).trigger( 'click' );
```

Comment:
On the `checkout.js` file, I used the `e` parameter and `$( this )` since this is the pattern used in the file.
Please check the [`remove_coupon`](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/client/legacy/js/frontend/checkout.js#L779-L783) function.